dlrs/main/classes/RollupServiceTest3.cls

```diff
@@ -113,6 +113,9 @@ private with sharing class RollupServiceTest3 {
     rollupSummary.CalculationMode__c = 'Scheduled';
     insert rollupSummary;
 
+    // burn the rollup cache so it can see the record
```

Comment:
Rollup config's were changed after the cache is build the first time. Clearing the cache allows the new rollups to be seen.